src/pages/User/components/UserHeader/styles.scss

```
@@ -0,0 +1,98 @@
+.UserHeader {
```

Please import the SCSS variables

```suggestion
@import 'src/css/app.scss';

.UserHeader {
```